extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -84,7 +64,7 @@ public function index() {
 		$this->response->setVal( 'item_placeholder', wfMessage( 'wikiacuratedcontent-content-item' ) );
 		$this->response->setVal( 'name_placeholder', wfMessage( 'wikiacuratedcontent-content-name' ) );
 
-		$itemTemplate = $this->sendSelfRequest( self::ITEM_FUNCTION_NAME )->toString();
+		$itemTemplate = $this->sendSelfRequest( 'item' )->toString();
```

Comment:
It's only function name - we usually just use raw string for that - it's a common pattern in our code.